.github/workflows/ci.yml

```
@@ -182,11 +236,210 @@ jobs:
+            platform=$(basename "$artifact" | sed 's/^cli-//')
+            if [ -f "$artifact/archlint" ]; then
+              cp "$artifact/archlint" "release/archlint-${platform}"
+              chmod +x "release/archlint-${platform}"
+            fi
+            if [ -f "$artifact/archlint.exe" ]; then
+              cp "$artifact/archlint.exe" "release/archlint-${platform}.exe"
+            fi
+          done
+
+      - name: Upload binaries to release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)